code/WorkInProgress/tarmStuff.dm

```diff
@@ -739,6 +739,9 @@ TYPEINFO(/obj/item/device/geiger)
 		. = ..()
 
 //DIRTY DIRTY PLAYERS
+TYPEINFO(/obj/submachine/laundry_machine/portable)
+	mats = list("telecrystal" = 5, "MET-2" = 15, "CON-1" = 10)
+
```

Comment:
until/unless I decide to actually add these as something other than admin-spawned stuff, I'd rather not have them mechscannable